docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  birthday-service:
+    image: birthday-service
+    build: .
+    env_file:
+      - ./app.env
+    ports:
+      - 8000:8000
+    depends_on:
+      - mongo
+      
+  mongo:
+    image: mongo
+    restart: always
+    ports:
+        - 27017:27017
+    environment:
+        MONGO_INITDB_ROOT_USERNAME: root
+        MONGO_INITDB_ROOT_PASSWORD: example
+    volumes:
+        - "./data/db:/data/db"
+    
+  mongo-express:
+    image: mongo-express
+    restart: always
+    ports:
+      - 8081:8081
+    environment:
+      ME_CONFIG_MONGODB_ADMINUSERNAME: root
+      ME_CONFIG_MONGODB_ADMINPASSWORD: example
+    depends_on:
+      - mongo
```

Comment:
Redundant

Reply:
i used express as a ui for the mongo data, but i'll remove it.